src/sentry/runner/commands/devserver.py

```diff
@@ -17,10 +17,11 @@
 @click.option('--reload/--no-reload', default=True, help='Autoreloading of python files.')
 @click.option('--watchers/--no-watchers', default=True, help='Watch static files and recompile on changes.')
 @click.option('--workers/--no-workers', default=False, help='Run asynchronous workers.')
+@click.option('--refresh/--no-refresh', default=False, help='Automatic browser refreshing on webpack builds')
```

Comment:
i didn't want to have reload and live reload. i could call it browser-reload or something